.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @WomenCodingCommunity @stepsen89 @lilyndk @dricazenck
```

Comment:
```suggestion
* @WomenCodingCommunity @Women-Coding-Community/frontend-team
```

Reply:
all are part of frontend-team, I guess will be easier to change via team, we can add people to the team and they can be automatically as owners.